qa/rpc-tests/feature_fedpeg.py

```diff
@@ -47,27 +49,61 @@ def __init__(self):
         self.setup_clean_chain = True
         self.num_nodes = 4
 
+    def add_options(self, parser):
+        parser.add_option("--parent_binpath", dest="parent_binpath", default="",
+                          help="Use a different binary for launching nodes")
+        parser.add_option("--parent_bitcoin", dest="parent_bitcoin", default=False, action="store_true",
+                          help="Parent nodes are Bitcoin")
+
+    def start_node(self, idx):
+        is_parent = idx < 2
+        binary = self.options.parent_binpath if is_parent and self.options.parent_binpath != "" else None
+        chain = "sidechain" if not is_parent else "regtest" if self.options.parent_bitcoin else "parent"
+        return start_node(idx, self.options.tmpdir, self.extra_args[idx], binary=binary, chain=chain)
+
     def setup_network(self, split=False):
+        if self.options.parent_bitcoin and self.options.parent_binpath == "":
+            raise "Can't run with --parent_bitcoin without specifying --parent_binpath"
```

Comment:
`Unexpected exception caught during testing: TypeError('exceptions must derive from BaseException',)`

Can we handle this?

Reply:
Just put the quote in `Exception()`